androidenhancedvideoplayer/src/main/java/com/profusion/androidenhancedvideoplayer/components/PlayerControls.kt

```diff
@@ -0,0 +1,86 @@
+package com.profusion.androidenhancedvideoplayer.components
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.material3.IconButton
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import com.profusion.androidenhancedvideoplayer.R
+
+@Composable
+fun PlayerControls(
+    isPlaying: Boolean,
+    onPreviousClick: () -> Unit,
+    onPauseToggle: () -> Unit,
+    onNextClick: () -> Unit,
+    decoration: ControlsDecoration,
+    modifier: Modifier = Modifier
+) {
+    Box(
+        contentAlignment = Alignment.CenterStart,
+        modifier = modifier
```

Comment:
IMO, not a good pattern. `modifier` is the go-to for changing style. If I call this control and set the `.background` on the modifier, I expect it to be used. However, this is not the case as it will be overwritten by `decoration.backgroundColor`.

I'd remove `decoration.backgroundColor` and just use the modifier we are receiving